prog/dftb+/prg_dftb/initprogram.F90

```diff
@@ -852,7 +852,7 @@ subroutine initProgramVariables(input, env)
     type(OVelocityVerlet), allocatable :: pVelocityVerlet
     type(OTempProfile), pointer :: pTempProfile
 
-    type(ORanlux), allocatable :: randomInit, randomThermostat
+    type(ORanlux), allocatable :: randomInit, randomThermostat, randomCharges, randomSpins
```

Comment:
I am somewhat undecided, whether we really need two more additional random generators, or whether this could be handled directly by `ṙandomInit` or by one common new random generator for both cases.

Reply:
Probably not needed. Depends on whether there is ever anything which non-deterministically pulls out random values from the generator but at the same time needs a repeatable random sequence for something else.
About the only thing I can think of that might do this isn't likely to be implemented (coupling from the past in MC).